### ISSUE TITLE
Add TOC to CONTRIBUTE.md

### ISSUE THREAD

**Issue by mik-laj** [apache/airflow-site#259]
Hello,

I will be happy when the Table of contents has been added to CONTRIBUTE.md. Its structure should be verified using pre-commit.

We can use
https://github.com/thlorenz/doctoc
Example integration with pre-commit:
https://github.com/apache/airflow/blob/0c1477f/.pre-commit-config.yaml#L130-L138

Best regards

**Comment by alok-kumar0421** [comment]
A pull request addressing this issue has been opened here: #1307.
Please let me know if any further adjustments or improvements are needed

**Comment by alok-kumar0421** [comment]
Created a follow-up improvement proposal here: #1308

**Comment by potiuk** [comment]
No.  do it straight away - it makes no sense to implement it manually and switch to automate after. All those issue creation here is completely unnecessary and useless and adds a lot of overheat - ask your agent to stop doing it.

**Comment by choo121600** [comment]
This has been resolved in #1354
so we can close this issue :)
